README.md

```diff
@@ -0,0 +1,343 @@
+-   Tutorial 4 - FAQ-style QA: [Jupyter notebook](https://github.com/deepset-ai/haystack/blob/master/tutorials/Tutorial4_FAQ_style_QA.ipynb)
+    or
+    [Colab](https://colab.research.google.com/github/deepset-ai/haystack/blob/master/tutorials/Tutorial4_FAQ_style_QA.ipynb)
+-   Tutorial 5 - Evaluation of the whole QA-Pipeline: [Jupyter noteboook](https://github.com/deepset-ai/haystack/blob/master/tutorials/Tutorial5_Evaluation.ipynb)
+    or
+    [Colab](https://colab.research.google.com/github/deepset-ai/haystack/blob/master/tutorials/Tutorial5_Evaluation.ipynb)
+-   Tutorial 6 - Better Retrievers via "Dense Passage Retrieval":
+    [Jupyter noteboook](https://github.com/deepset-ai/haystack/blob/master/tutorials/Tutorial6_Better_Retrieval_via_DPR.ipynb)
+    or
+    [Colab](https://colab.research.google.com/github/deepset-ai/haystack/blob/master/tutorials/Tutorial6_Better_Retrieval_via_DPR.ipynb)
+
+## Quick Tour
+[File Conversion](https://github.com/deepset-ai/haystack/blob/master/README.md#1-file-conversion) | [Preprocessing](https://github.com/deepset-ai/haystack/blob/master/README.md#2-preprocessing) | [DocumentStores](https://github.com/deepset-ai/haystack/blob/master/README.md#3-documentstores) | [Retrievers](https://github.com/deepset-ai/haystack/blob/master/README.md#5-retrievers) | [Readers](https://github.com/deepset-ai/haystack/blob/master/README.md#5-readers) | [REST API](https://github.com/deepset-ai/haystack/blob/master/README.md#6-rest-api) |  [Labeling Tool](https://github.com/deepset-ai/haystack/blob/master/README.md#7-labeling-tool) 
+
+### 1) File Conversion
+**What**  
+Different converters to extract text from your original files (pdf, docx, txt, html).
+While it's almost impossible to cover all types, layouts and special cases (especially in PDFs), we cover the most common formats (incl. multi-column) and extract meta information (e.g. page splits). The converters are easily extendable, so that you can customize them for your files if needed.
+
+**Available options**  
+- Txt
+- PDF 
+- Docx
+- Apache Tika (Supports > 340 file formats)
+
+**Example**  
+
+```python
+#PDF
+from haystack.file_converter.pdf import PDFToTextConverter
+converter = PDFToTextConverter(remove_numeric_tables=True, valid_languages=["de","en"])
+doc = converter.convert(file_path=file, meta=None)
+# => {"text": "text first page \f text second page ...", "meta": None}
+
+#DOCX
+from haystack.file_converter.docx import DocxToTextConverter
+converter = DocxToTextConverter(remove_numeric_tables=True, valid_languages=["de","en"])
+doc = converter.convert(file_path=file, meta=None)
+# => {"text": "some text", "meta": None}
+```
+
+
+### 2) Preprocessing
+**What**  
+Cleaning and splitting of your texts are crucial steps that will directly impact the speed and accuracy of your search.
+Especially the splitting of larger texts is key to achieve fast query speed. The longer the texts that the retriever passes to the reader, the slower your queries. 
```

Comment:
My attempt to make this line sound a little more natural:
"The splitting of larger texts is especially important for achieving fast query speed."